app/Http/Controllers/RecurringController.php

```diff
@@ -7,8 +7,20 @@
 use App\Recurring;
 use Auth;
 use App\Jobs\ProcessRecurrings;
+use Illuminate\Support\Facades\Validator;
 
 class RecurringController extends Controller {
+    protected function validator(array $data)
+    {
```

Comment:
Next time, put the `{` on the same line as the definition of the function. I know, PSR-2, but we don't (yet) use PSR-2.